R/doc-tidy-selection.R

```diff
@@ -0,0 +1,35 @@
+#' Argument type: tidy-select
+#'
+#' @description
+#' This page describes the `<tidy-select>` argument modifier which indicates
+#' the argument supports **tidy selections**. Tidy selection provides a concise
+#' dialect of R for selecting variables based on their names or properties.
+#'
+#' Tidy selection is a variant of tidy evaluation. This means that inside
+#' functions, tidy-select arguments require special attention, as described in
```

Comment:
```suggestion
#' functions that you create, tidy-select arguments require special attention, as described in
```

Right? Even though we got to this page from, say, `dplyr::select()`'s documentation of the `...`, you are talking about the case where the reader of the documentation makes _their own wrapper_ function, right? So we should just be super clear about that - because you can also come to this page when you're just like "what's tidyselect?" and you aren't interested in these advanced ideas.

Maybe this whole paragraph should move down into `# Details` or `# Wrapping tidyselect arguments`, with `## Indirection` as a subheader?

That would let you provide the main details up top (i.e. what is tidyselect) and then later on talk about the complex stuff